### ISSUE TITLE
(3/5) Configure the gcpFilestoreBackups helm chart to use chartpress

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/infrastructure#4703]
### Context

[chartpress](https://github.com/jupyterhub/chartpress) is a useful tool for automatically building and publishing helm charts for use downstream. Let's setup chartpress for the gcpFilestoreBackups project so that our downstream 2i2c infrastructure can make use of it.

### Definition of Done

- [ ] chartpress automatically builds and publishes the gcpFilestoreBackups helm chart for downstream use

**Comment by sgibson91** [comment]
Need to figure out how/where to publish the helm chart to.

Firstly I tried copying jupyterhub's gh-pages approach, but formatting the website felt like too much effort.

So then I moved onto trying to use GHCR like [Tarashish has](https://github.com/sunu/jupyter-home-nfs/blob/c9d7b89b688e8b474358806ea1ceb46c0dbb9b15/.github/workflows/build-publish-docker-helm.yaml#L53-L55), but receiving `Error: invalid_reference: invalid repository` errors.
